Dockerfile

```
@@ -29,10 +29,11 @@ WORKDIR /opt/mesa
 
 COPY . /opt/mesa
 
-RUN cd / && git clone https://github.com/projectmesa/mesa-examples.git
-
 EXPOSE 8765/tcp
 
-RUN pip3 install -e /opt/mesa
+RUN pip3 install -e /opt/mesa[all]
```

I think `rec` is enough here, since `all` also installs dev dependencies and such.

```suggestion
RUN pip3 install -e /opt/mesa[rec]
```

---

I tested rec and it works.